src/main/java/org/jenkinsci/plugins/periodicbackup/PeriodicBackupPlugin.java

```diff
@@ -81,7 +83,7 @@ private void updateDebugLogger(final Level level, final String... debugLoggerNam
         }
 
         if (target == null) {
-          recorder.targets.add(new Target(current, level));
+          recorder.getLoggers().add(new Target(current, level));

```

Comment:
FTR this change allows the plugin to work again in 2.509+, it's currently broken due to https://github.com/jenkinsci/jenkins/pull/10612.